src/components/calcite-flow-item/calcite-flow-item.scss

```diff
@@ -2,11 +2,11 @@
   background-color: var(--calcite-app-background-content);
   display: flex;
   height: 100%;
+  width: 100%;
```

Comment:
display flex should act like a block container but apply flexbox model to its children. It shouldn't need width: 100% to stretch to fill its container if its behaving like a block.

Reply:
I opened a PR to address some of this stuff. This could follow up merging of this PR if desired. If so lmk and I'll adjust to approve. https://github.com/Esri/calcite-app-components/pull/669